Alignment/APEEstimation/test/autoSubmitter/autoSubmitter.py

```diff
@@ -1,5 +1,5 @@
 from __future__ import print_function
-import ConfigParser
+import configparser as ConfigParser
```

Comment:
@mteroerd can you please cross-check this change if this is still working for you? 
Thx.

Reply:
Even if Alignment/APEEstimation/test/autoSubmitter/config.ini is just a template, it should be enough to test the parsing of the INI file.  
I tried with `python autoSubmitter.py -c config.ini` there are no crash due to ConfigParser.
If you have a real config.ini file, I can try again with that.

Reply:
@mmusich I tried one config file and it seems to work just fine.

Reply:
@mteroerd also in a CMSSW version compiled with Py3 e.g.: `CMSSW_11_1_PY3_X_2020-01-28-2300`? thanks!

Reply:
@mmusich When running in that CMSSW version, I get an error, which however is not because of this change. It is because of the enableCAF function call that is done. It executes /usr/share/Modules/init/python.py, which produces a syntax error as in that file there is an exec statement without parantheses.
I get other errors from using "has_key" on dictionaries instead of "x in dictionary", which apparently is not allowed in Py3 anymore.  

However, the configparser itself seems to work fine.

Reply:
@mteroerd thanks for the investigation.
We eventually need to migrate all tools to be Py3 compliant, see issue: https://github.com/cms-sw/cmssw/issues/28830.
When you have time would be available to try to migrate the APE submission code to be Py2 and Py3 compliant? Thanks!